DataCollector/RedisDataCollector.php

```diff
@@ -42,7 +42,7 @@ public function __construct(RedisLogger $logger)
     public function collect(Request $request, Response $response, \Exception $exception = null)
     {
         $this->data = array(
-            'commands' => null !== $this->logger ? $this->logger->getCommands() : array(),
```

Comment:
The `$logger` is a hard dependency, see 

https://github.com/snc/SncRedisBundle/blob/c5aaf5165301c0a5ad1f7af878512a7918ea2a96/DataCollector/RedisDataCollector.php#L34-L37

This condition only makes sense for scenarios where the constructor is not invoked:

* instantiated via reflection
* collector is overridden, but original constructor is not invoked